.travis.yml

```diff
@@ -1,20 +1,38 @@
 language: php
 
+sudo: required
+
+services:
+  - docker
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
+env:
+  global:
+    - PHP_AMQP_VERSION=v1.7.0 LIBRABBITMQ_VERSION=v0.8.0
+  matrix:
+    - DEPENDENCIES=""
+    - DEPENDENCIES="--prefer-lowest --prefer-stable"

```

Comment:
I don't think that this is necessary. It only spawns more build processes and spams travis. I've never seen projects that do this.